test/integration/cors_filter_integration_test.cc

```diff
@@ -20,30 +20,21 @@ class CorsFilterIntegrationTest : public HttpIntegrationTest,
 protected:
   void testPreflight(Http::TestHeaderMapImpl&& request_headers,
                      Http::TestHeaderMapImpl&& expected_response_headers) {
-    executeActions({
-        [&]() -> void { codec_client_ = makeHttpConnection(lookupPort("http")); },
-        [&]() -> void { codec_client_->makeHeaderOnlyRequest(request_headers, *response_); },
-        [&]() -> void { response_->waitForEndStream(); },
-        [&]() -> void { cleanupUpstreamAndDownstream(); },
-    });
-
+    initialize();
+    codec_client_ = makeHttpConnection(lookupPort("http"));
+    codec_client_->makeHeaderOnlyRequest(request_headers, *response_);
+    response_->waitForEndStream();
     EXPECT_TRUE(response_->complete());
     compareHeaders(response_->headers(), expected_response_headers);
   }
 
   void testNormalRequest(Http::TestHeaderMapImpl&& request_headers,
                          Http::TestHeaderMapImpl&& expected_response_headers) {
-    executeActions({
-        [&]() -> void { codec_client_ = makeHttpConnection(lookupPort("http")); },
-        [&]() -> void {
-          sendRequestAndWaitForResponse(request_headers, 0, expected_response_headers, 0);
-        },
-        [&]() -> void { response_->waitForEndStream(); },
-        [&]() -> void { cleanupUpstreamAndDownstream(); },
```

Comment:
It seems in some places that cleanup has been dropped, but in others it's still around. Can you talk about what the guiding principle here is?

Reply:
I think I missed one call in proto_integration_test.cc which I've fixed.  Please point out others you see.